js/foam/apps/calc/SecondaryButtonsView.js

```diff
@@ -29,28 +29,28 @@ CLASS({
           <div id="%%id" class="buttons button-row secondaryButtons">
             <div class="button-column" style="flex-grow: 1;-webkit-flex-grow: 1;">
               <div class="button-row">
```

Comment:
This changes the way navigation works as the old one relies on finding elements via aria labels in english, this update now makes the navigation work for all languages.

Reply:
Nice! :)